sympy/matrices/tests/test_matrices.py

```diff
@@ -2325,14 +2325,12 @@ def test_replace_map():
     assert N == K
 
 def test_atoms():
-    from sympy.abc import x
     m = Matrix([[1, 2], [x, 1 - 1/x]])
     assert m.atoms() == set([S(1),S(2),S(-1), x])
     assert m.atoms(Symbol) == set([x])
 
 @slow
 def test_pinv():
-    from sympy.abc import a, b, c, d
```

Comment:
... it's okay to import locally.
It's not really important either way though. The knee-jerk reaction is "avoid gratuitious change because we want to be able to pinpoint semantic changes in the history", but then imports are mere housekeeping so that particular history isn't very important.